interface/orders/procedure_provider_edit.php

```diff
@@ -144,6 +170,7 @@ function invalue($name)
         <div class="row">
             <div class="col-sm-12">
                 <form method='post' name='theform' action='procedure_provider_edit.php?ppid=<?php echo attr_url($ppid) ?>'>
```

Comment:
note also need to protect csrf for POST since this form gets submitted as a post, which will add above